packages/blocks/src/__internal__/utils/std.ts

```diff
@@ -90,3 +90,28 @@ export const throttle = <T extends (...args: any[]) => void>(
     timer = setTimeout(setTimer, limit);
   } as T;
 };
+
+/**
+ * This function takes a value value, a minimum value min, and a maximum value max,
+ * and returns the value of value clamped to the range [min, max].
+ *
+ * This means that if value is less than min, the function will return min;
+ * if value is greater than max, the function will return max;
+ * otherwise, the function will return value.
+ *
+ * @example
+ * ```ts
+ * const x = clamp(10, 0, 5); // x will be 5
+ * const y = clamp(3, 0, 5); // y will be 3
+ * const z = clamp(-1, 0, 5); // z will be 0
+ * ```
+ */
+export const clamp = (value: number, min: number, max: number): number => {
```

Comment:
I think we prefer `function` over `const` in util modules. This doesn't matter, though.